packages/databyss-ui/primitives/Button/SidebarButton.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { variant, flexbox, compose, border, color } from 'styled-system'
+import { BaseControl, Text } from '../'
+// HACK: if View is imported from '../' above, it breaks storybook:build (reason unknown)
+import View from '../View/View'
+import styled from '../styled'
+import buttons from '../../theming/buttons'
+
+const borderVariant = variant({
+  prop: 'borderVariant',
+  scale: 'borderVariants',
+  variants: {
+    // need one member to enable theming
+    default: {},
+  },
+})
+
+const variants = variant({
+  prop: 'variant',
+  scale: 'buttonVariants',
+})
+
+const StyledView = styled(
+  View,
+  compose(
+    flexbox,
+    border,
+    variants,
+    borderVariant,
+    color
+  )
+)
+
+const SidebarButton = ({ onPress, variant, children, ...others }) => {
+  let _children = children
+  const { buttonVariants } = buttons
+
+  if (typeof children === 'string') {
+    _children = (
+      <Text variant="uiTextExtraSmall" color={buttonVariants[variant].color}>
+        {children}
+      </Text>
+    )
+  }
+  return (
+    <BaseControl
+      onPress={onPress}
+      rippleColor={buttonVariants[variant].rippleColor}
+      borderRadius="50%"
```

Comment:
This has to be an Int or Float for native. So use `pxUnits()` and set it to half the height.
```
borderRadius={pxUnits(space.editorBlockMenuHeight / 2)}
width={pxUnits(space.editorBlockMenuHeight)}
height={pxUnits(space.editorBlockMenuHeight)}
```